projects/daily_dribble_newsletter/styles.css

```diff
@@ -0,0 +1,53 @@
+.body {
+    /** background-color: skyblue; */
+    height: auto;
+    border: 3px solid black;
+    text-align: center;
+    margin: 10px 350px;
```

Comment:
I don't see the use of this. It makes the content not look good on smaller screens.